src/CMakeLists.txt

```diff
@@ -88,11 +87,9 @@ else()
             ${KERNEL_SRC}
             "kernel/accumulate_maxsumexp/cuda.cu"
             "kernel/add/cuda.cu"
-            "kernel/add_slice_inplace/cuda.cu"
```

Comment:
some sources are deleted. I am sure you did something very strange with git. You forked the repository and then uploaded old version of some files on top of cloned repo. This is not how you work with git.